mpxe/integration_tests/int_test.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+import time
+from mpxe.harness import pm
+from mpxe.harness import project
```

Comment:
You don't use the `time` or `project` imports, you might also consider importing `ProjectManager` directly instead of importing `pm`